src/styles/main.scss

```diff
@@ -0,0 +1,68 @@
+.body {
+  position: relative;
+  height: 100vh;
+  margin: 0 auto;
+}
+
+.clock {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  &__circle {
+    position: absolute;
+    z-index: 3;
+    height: 20px;
+    width: 20px;
+    border-radius: 50%;
+    background-color: #14425a;
+  }
+
+  &__minute {
+    position: absolute;
+    top: -90px;
+    left: 5px;
+    z-index: 2;
+    width: 10px;
+    height: 100px;
+    border-radius: 25px;
+    background-color: #14425a;
+    transition: transform 3s;
+    transform-origin: bottom;
+    animation: second 60s linear infinite;
+  }
+
+  &__second {
+    position: absolute;
+    top: -40px;
+    left: 5px;
+    z-index: 1;
+    height: 50px;
+    width: 10px;
+    background-color: #e37448;
+    border-radius: 25px;
+    transform-origin: bottom;
+    animation: minute 3600s linear 60s infinite;
+  }
+}
+
+@keyframes second {
+  0% {
+    transform: rotate(0);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes minute {
+  0% {
+    transform: rotate(0);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
There's no difference between these keyframes. Just name it "fullRotate" or something like this.